app/home.tmpl

```diff
@@ -70,9 +71,17 @@
         .row > div:nth-of-type(3n+3) div[data-name]::before {
             background: var(--tile-3-bgcolor);
         }
+        
+        /* Enable darkmode in a Bootstrap Page */
+        @media (prefers-color-scheme: dark) {
```

Comment:
Dark theme looks great, but footer should be dark too.
I've made some tests and this rule should do the work 
```
footer .text-muted {
    color: rgba(var(--bs-secondary-rgb),var(--bs-text-opacity))!important;
}
```

You should also remove `bg-light` class on footer element